lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -5,7 +5,7 @@ module Billing #:nodoc:
     # This gateway uses the current Stripe {Payment Intents API}[https://stripe.com/docs/api/payment_intents].
     # For the legacy API, see the Stripe gateway
     class StripePaymentIntentsGateway < StripeGateway
-      self.supported_countries = %w(AT AU BE BG BR CA CH CY CZ DE DK EE ES FI FR GB GR HK IE IT JP LT LU LV MT MX NL NO NZ PL PT RO SE SG SI SK US)
+      self.supported_countries = %w(AE AT AU BE BG BR CA CH CY CZ DE DK EE ES FI FR GB GR HK HU IE IN IT JP LT LU LV MT MX MY NL NO NZ PL PT RO SE SG SI SK US)
```

Comment:
Suggestion 💬 :

Given the fact that StripePaymentINtentsGateway inherits from StripeGateway and the list of countries is the same, we can remove this line and avoid duplication.